src/schema/objects/columns.yaml

```
@@ -0,0 +1,507 @@
+  - type: number
+handedness:
+  name: handedness
+  description: |
+    String value indicating one of "left", "right", "ambidextrous".
+
+    For "left", use one of these values: `left`, `l`, `L`, `LEFT`, `Left`.
+
+    For "right", use one of these values: `right`, `r`, `R`, `RIGHT`, `Right`.
+
+    For "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`, `AMBIDEXTROUS`,
+    `Ambidextrous`.
+  type: string
+  enum:
+  - left
+  - l
+  - L
+  - LEFT
+  - Left
+  - right
+  - r
+  - R
+  - RIGHT
+  - Right
+  - ambidextrous
+  - a
+  - A
+  - AMBIDEXTROUS
+  - Ambidextrous
+  - n/a
+hemisphere:
+  name: hemisphere
+  description: |
+    The hemisphere in which the electrode is placed, one of `['L' or 'R']` (use capital).
+  type: string
+  enum:
+  - L
+  - R
+high_cutoff:
+  name: high_cutoff
+  description: |
+    Frequencies used for the low-pass filter applied to the channel in Hz.
+    If no low-pass filter applied, use `n/a`.
+    Note that hardware anti-aliasing in A/D conversion of all MEG/EEG electronics
+    applies a low-pass filter; specify its frequency here if applicable.
+  anyOf:
+  - type: number
+    unit: Hz
+    minimum: 0
+  - type: string
+    enum:
+    - n/a
+hplc_recovery_fractions:
+  name: hplc_recovery_fractions
+  description: |
+    HPLC recovery fractions (the fraction of activity that gets loaded onto the HPLC).
+  type: number
+  unit: arbitrary
+impedance:
+  name: impedance
+  description: |
+    Impedance of the electrode, units MUST be in `kOhm`.
+  type: number
+  unit: kOhm
+index:
+  name: index
+  description: |
+    The label integer index.
+  type: integer
+low_cutoff:
+  name: low_cutoff
+  description: |
+    Frequencies used for the high-pass filter applied to the channel in Hz.
+    If no high-pass filter applied, use `n/a`.
+  anyOf:
+  - type: number
+    unit: Hz
+  - type: string
+    enum:
+    - n/a
+manufacturer:
+  name: manufacturer
+  description: |
+    The manufacturer for each electrode.
+    Can be used if electrodes were manufactured by more than one company.
+  type: string
+mapping:
+  name: mapping
+  description: |
+    Corresponding integer label in the standard BIDS label lookup.
+  type: integer
+material:
+  name: material
+  description: |
+    Material of the electrode (for example, `"Tin"`, `"Ag/AgCl"`, `"Gold"`).
+  type: string
+metabolite_parent_fraction:
+  name: metabolite_parent_fraction
+  description: |
+    Parent fraction of the radiotracer (0-1).
+  type: number
+  minimum: 0
+  maximum: 1
+metabolite_polar_fraction:
+  name: metabolite_polar_fraction
+  description: |
+    Polar metabolite fraction of the radiotracer (0-1).
+  type: number
+  minimum: 0
+  maximum: 1
+name_channels:
+  name: name
+  description: |
+    Label of the channel.
+    The label must correspond to `_electrodes.tsv` name and all ieeg type channels are
+    required to have a position.
+    The reference channel name MAY be provided in the reference column.
```

make obvious that this is a standardized column name

```suggestion
    The reference channel name MAY be provided in the `reference` column.
```